pb-cli.js

```diff
@@ -418,7 +449,24 @@ var talkResp = function (error, response, body) {
     }
 }
 
-var chatResp = function(error, response, body) {
+var writeLogFile = (input, output) => {
+    var arr = [];
+    var note = {
+      input,
+      output
+    };
+    try {
+      var resLogString = fs.readFileSync('Response-log.json');
```

Comment:
Most users are probably not expecting logs as JSON - it could be useful to give them the option of receiving logs in a JSON format, but we should write to a TSV file by default so the logs can be imported into a spreadsheet easily. 😄 

Also a nitpick - naming the log file with a capital letter is a break from convention. Usually only Class constructors start with a capital letter.